app-backend/backsplash-core/src/main/scala/com/rasterfoundry/backsplash/package.scala

```diff
@@ -1,8 +1,21 @@
 package com.rasterfoundry
 
+import java.net.{URI, URL}
+import java.nio.charset.Charset
+import java.nio.file.Paths
+
 import cats.effect._
+import com.amazonaws.services.s3.{AmazonS3ClientBuilder, AmazonS3URI}
 import com.colisweb.tracing.TracingContext
+import geotrellis.contrib.vlm.config.{HdfsConfig, S3Config}
+import geotrellis.spark.io.hadoop.HdfsRangeReader
```

Comment:
Do we plan to support the full range of sources here? I mean we need the spark dependency for S3RangeReader anwyay, so it doesn't make too much of a difference

Reply:
This was a straight copy/paste job from GT since this function wasn't readily accessible -- we'll be able to remove it once we upgrade it.

I'll adjust it down though because you're right